File: problem-2/main.c
#include <stdio.h>

int main() {
    int a=1,b=2,sum=0;
    while (a<4000000){
        if(a%2==0) {
            sum+=a;
        }
    int swap = a;
    a = b;
    b = swap + b;
    }
    printf("%d\n",sum);
    return 0;
}

File: problem-2/main.go
package main
import "fmt"

func main() {
    a,b:=1,2
    sum:=0
    for a<4000000 {
        if a%2==0 {
            sum+=a
        }
        a,b=b,a+b
    }
    fmt.Println(sum)
}

